src/plugins/home/server/tutorials/instructions/functionbeat_instructions.ts

```diff
@@ -320,8 +320,11 @@ export function functionbeatAWSInstructions() {
   });
   const defaultPost = i18n.translate('home.tutorials.common.functionbeatAWSInstructions.textPost', {
     defaultMessage:
-      'Where `<your-access-key>` and `<your-secret-access-key>` are your account credentials and \
-`us-east-1` is the desired region.',
+      'Where {accessKey} and {secretAccessKey} are your account credentials and `us-east-1` is the desired region.',
+    values: {
+      accessKey: '`<your-access-key>`',
+      secretAccessKey: '`<your-secret-access-key>`',
+    },
```

Comment:
Idem, could we use `ignoreTag` here?

Reply:
Added to the PR docs. If its HTML tags, it is preferred not to ignore the Tag to have some string verification that the html tags you are adding are properly formatted and closed.
If it is not an HTML tag and it is just a fomat preference (like in the case above) then ignoreTag makes sense